### PR TITLE
Fix for Django 2.0

### PR DESCRIPTION
**Pull request by morlandi**
User.is_authenticated is now an attribute; backwards-compatibility support for using it as a method has been removed in Django 2.0.
See:

https://docs.djangoproject.com/en/1.11/ref/contrib/auth/#django.contrib.auth.models.User.is_authenticated

Also, unit tests have been revised, and Django 2.0 added to Tox envlist

### CODE REVIEW

**Comment by jaap3** [comment]
Thank you! I think we can drop support for Django < 1.11 for the next release. Django 1.8 will not be supported after ~April 2018 and older versions of smuggler will work just fine.

Can you update your PR so it only works on Django>=1.11? This means removing the compatibility shim, updating the tox config and removing the conditional imports. 

If any conditional imports are still necessary please change them so the "new" import is tried first and the "old" import is in the except clause as a fallback.

Your contribution is much appreciated!

**Comment by morlandi** [comment]
@jaap3 thanks for clarifications; I'm willing to improve my PR as suggested, and plan to do it in a week or so

**Comment by morlandi** [comment]
@jaap3 , I was finally able to include your suggestions into my PR.

Dropping support fro Django < 1.11, nor conditional imports neither compatibility shims are needed any more, so the resulting code is cleaner.
Test coverage recovers as well.

For some reason, travis keeps running checks for Django 1.8 and 1.9 configurations, even if I removed them from tox.ini. That's the reason why some tests fail.

Thank you in advance for reviewing my PR.
Please feel free to give advices on how to improve it as needed.

Mario

**Comment by jaap3** [comment]
Thank you!

File: README.rst [hunk 1]
```diff
@@ -131,6 +131,12 @@ Load form (with ``SMUGGLER_FIXTURE_DIR`` configured):
 Release notes
 =============
 
+Version 0.9.0 (2018-??-??)
```
Comment (semente) [inline README.rst]: Hey @jaap3, I was thinking in releasing version 0.8 as 1.0 (bug fixes if we ever need on 1.0 branch, then minor bugfixes releases on 1.0.x).

The current master we release as 1.1.

What do you think?

File: .travis.yml [hunk 2]
```diff
@@ -3,13 +3,19 @@ language: python
 python:
   - 2.7
```
Comment (semente) [inline .travis.yml]: Django 2 supports Python 3 only, we can take this out.

File: README.rst (hunk 1 above)
Comment (semente) [reply README.rst]: Actually, better we release it as 0.9. We let it mature than tag it as 1.0.

If we ever add new features at futre, it should go to 1.1 and the current 0.9.x tag turns in 1.0.

**Comment by semente** [comment]
@morlandi @jaap3 awesome! everything looks great! I'm releasing 0.9
Thanks!

**Comment by semente** [comment]
@morlandi you have now smuggler 0.9.0 on PyPI :)

**Comment by morlandi** [comment]
Yes sir ! thankyou

**Pull request merged**